hieradata/class/mongo.yaml

```diff
@@ -10,6 +10,7 @@ lv:
     pv:
       - '/dev/sdb1'
       - '/dev/sdd1'
+      - '/dev/sdf1/'
```

Comment:
Can we remove the extra slash at the end of the string so it is consistent.

Reply:
thank you @dwhenry . good spot. I have now corrected this.